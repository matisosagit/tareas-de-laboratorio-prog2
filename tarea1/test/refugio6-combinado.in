crearRefugio

# se crean 10 Perros para ingresarlos a la Refugio

crearFecha 18/12/2020
crearPerro 1 Daisy 5 9 Beagle curiosa y aventurera que siempre sigue su olfato a nuevas experiencias
agregarEnRefugio

crearFecha 10/04/2015
crearPerro 2 Bobby 9 5 Schnauzer anciano pero sabio que disfruta de la tranquilidad y los paseos cortos
agregarEnRefugio

crearFecha 03/08/2021
crearPerro 3 Milo 4 10 Dálmata enérgico con un amor por correr y jugar con otros perros
agregarEnRefugio

crearFecha 27/09/2020
crearPerro 4 Chloe 5 6 Pomerania esponjosa que adora ser el centro de atención
agregarEnRefugio

crearFecha 17/07/2021
crearPerro 5 Zeus 4 8 Pastor alemán valiente y bien entrenado que trabaja como perro guardián
agregarEnRefugio

crearFecha 09/03/2023
crearPerro 6 Lola 2 7 Shih tzu consentida que ama estar en brazos y recibir mimos
agregarEnRefugio

crearFecha 17/07/2021
crearPerro 7 Bruno 5 9 Boxer juguetón que nunca se cansa de saltar y hacer travesuras
agregarEnRefugio

crearFecha 06/01/2020
crearPerro 8 Maya 4 6 Akita inu independiente pero muy leal
agregarEnRefugio

crearFecha 13/09/2015
crearPerro 9 Canela 9 5 Mestiza dulce y tranquila que disfruta de las siestas al sol
agregarEnRefugio

crearFecha 11/11/2022
crearPerro 10 Otto 3 7 San Bernardo gigante pero bonachón que adora la nieve y los abrazos
agregarEnRefugio

imprimirRefugio

removerDeRefugio 8
removerDeRefugio 10
removerDeRefugio 4

imprimirRefugio

# Hay dos Perros fecha

ingresaronPerrosFechaRefugio 17/7/2021

# No hay Perros fecha

ingresaronPerrosFechaRefugio 15/2/2023

liberarRefugio

Fin