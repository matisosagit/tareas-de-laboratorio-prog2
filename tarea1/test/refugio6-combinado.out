1>2>#  se crean 10 Perros para ingresarlos a la Refugio.
3>4>5>6>7>8>9>10>11>12>13>14>15>16>17>18>19>20>21>22>23>24>25>26>27>28>29>30>31>32>33>Perro 2
Nombre: Bobby
Edad: 9
Descripcion:  Schnauzer anciano pero sabio que disfruta de la tranquilidad y los paseos cortos
Fecha de ingreso: 10/4/2015
Vitalidad: 5
Perro 9
Nombre: Canela
Edad: 9
Descripcion:  Mestiza dulce y tranquila que disfruta de las siestas al sol
Fecha de ingreso: 13/9/2015
Vitalidad: 5
Perro 8
Nombre: Maya
Edad: 4
Descripcion:  Akita inu independiente pero muy leal
Fecha de ingreso: 6/1/2020
Vitalidad: 6
Perro 4
Nombre: Chloe
Edad: 5
Descripcion:  Pomerania esponjosa que adora ser el centro de atención
Fecha de ingreso: 27/9/2020
Vitalidad: 6
Perro 1
Nombre: Daisy
Edad: 5
Descripcion:  Beagle curiosa y aventurera que siempre sigue su olfato a nuevas experiencias
Fecha de ingreso: 18/12/2020
Vitalidad: 9
Perro 5
Nombre: Zeus
Edad: 4
Descripcion:  Pastor alemán valiente y bien entrenado que trabaja como perro guardián
Fecha de ingreso: 17/7/2021
Vitalidad: 8
Perro 7
Nombre: Bruno
Edad: 5
Descripcion:  Boxer juguetón que nunca se cansa de saltar y hacer travesuras
Fecha de ingreso: 17/7/2021
Vitalidad: 9
Perro 3
Nombre: Milo
Edad: 4
Descripcion:  Dálmata enérgico con un amor por correr y jugar con otros perros
Fecha de ingreso: 3/8/2021
Vitalidad: 10
Perro 10
Nombre: Otto
Edad: 3
Descripcion:  San Bernardo gigante pero bonachón que adora la nieve y los abrazos
Fecha de ingreso: 11/11/2022
Vitalidad: 7
Perro 6
Nombre: Lola
Edad: 2
Descripcion:  Shih tzu consentida que ama estar en brazos y recibir mimos
Fecha de ingreso: 9/3/2023
Vitalidad: 7
34>El perro con id 8 se removió del refugio.
35>El perro con id 10 se removió del refugio.
36>El perro con id 4 se removió del refugio.
37>Perro 2
Nombre: Bobby
Edad: 9
Descripcion:  Schnauzer anciano pero sabio que disfruta de la tranquilidad y los paseos cortos
Fecha de ingreso: 10/4/2015
Vitalidad: 5
Perro 9
Nombre: Canela
Edad: 9
Descripcion:  Mestiza dulce y tranquila que disfruta de las siestas al sol
Fecha de ingreso: 13/9/2015
Vitalidad: 5
Perro 1
Nombre: Daisy
Edad: 5
Descripcion:  Beagle curiosa y aventurera que siempre sigue su olfato a nuevas experiencias
Fecha de ingreso: 18/12/2020
Vitalidad: 9
Perro 5
Nombre: Zeus
Edad: 4
Descripcion:  Pastor alemán valiente y bien entrenado que trabaja como perro guardián
Fecha de ingreso: 17/7/2021
Vitalidad: 8
Perro 7
Nombre: Bruno
Edad: 5
Descripcion:  Boxer juguetón que nunca se cansa de saltar y hacer travesuras
Fecha de ingreso: 17/7/2021
Vitalidad: 9
Perro 3
Nombre: Milo
Edad: 4
Descripcion:  Dálmata enérgico con un amor por correr y jugar con otros perros
Fecha de ingreso: 3/8/2021
Vitalidad: 10
Perro 6
Nombre: Lola
Edad: 2
Descripcion:  Shih tzu consentida que ama estar en brazos y recibir mimos
Fecha de ingreso: 9/3/2023
Vitalidad: 7
38>#  Hay dos Perros fecha.
39>Perro 5
Nombre: Zeus
Edad: 4
Descripcion:  Pastor alemán valiente y bien entrenado que trabaja como perro guardián
Fecha de ingreso: 17/7/2021
Vitalidad: 8
Perro 7
Nombre: Bruno
Edad: 5
Descripcion:  Boxer juguetón que nunca se cansa de saltar y hacer travesuras
Fecha de ingreso: 17/7/2021
Vitalidad: 9
40>#  No hay Perros fecha.
41>No se encontraron perros ingresados en la fecha determinada.
42>43>Fin.
